svsim/package.mill

```diff
@@ -25,4 +25,14 @@ trait Svsim extends CrossSbtModule with HasCommonOptions with ScalafmtModule {
   object test extends SbtTests with TestModule.ScalaTest with ScalafmtModule {
     def ivyDeps = Agg(v.scalatest, v.scalacheck)
   }
+
+  def clangformat = Task {
+    os.proc(
+      "clang-format",
+      "-i",
+      "--verbose",
+      s"--style=file:${os.pwd / os.RelPath("svsim/.clang-format")}",
+      os.pwd / os.RelPath("svsim/src/main/resources/simulation-driver.cpp")
+    ).call(cwd = Task.dest)
+  }
```

Comment:
```suggestion
  def clangFormatConfig = Task.Source { os.pwd / "svsim" / ".clang-format" }
  
  def cppSources = Task.Sources {
    Agg(os.pwd / "svsim/src/main/resources/simulation-driver.cpp")
  }

  def clangformat: Command[Unit] = Task.Command {
    os.proc(
      "clang-format",
      "-i",
      "--verbose",
      s"--style=file:${clangFormatConfig())}",
      cppSources(): _*
    ).call(cwd = Task.dest)
  }
```
I don't think this diff is exactly right as is but something like the above

A couple of minor suggestions, use `Task.Source[s]` for source files, make the clangFormat def a Command instead of just Task (which Mill might try to cache).

Reply:
I think I have this working in: https://github.com/chipsalliance/chisel/pull/4765/commits/875680d57fc2c35cadeb80f50c5ed74238eda03e